views/playground/responsive-utility-classes/index.html.twig

```diff
@@ -0,0 +1,70 @@
+{% extends '@pulsar/pulsar/layouts/base.html.twig' %}
```

Comment:
Will add a new ticket for a utilities section in the lexicon/gitbook. Will need some refactoring of blocks due to non inheritance of the javascript block in tabs